### PR TITLE
2377: Edit Case modal and endpoints for updating case status and caption

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2019-11-06 at 3 23 30 PM

### CODE REVIEW

File: web-client/src/styles/forms.scss [hunk 1]
```diff
@@ -160,6 +160,10 @@ textarea.caption {
   margin-bottom: 5px;
 }
 
+select.case-status {
+  width: 379px;
```
Comment (codyseibert) [inline web-client/src/styles/forms.scss:164]: this seems a bit arbitrary.  does width 100% not work?  I guess I'm just wondering what issue this solved

**Review by codyseibert — APPROVED** [review]

File: web-client/src/styles/forms.scss (hunk 1 above)
Comment (matthopson) [reply web-client/src/styles/forms.scss:164]: To match the caption textarea
https://github.com/flexion/ef-cms/blob/develop/web-client/src/styles/forms.scss#L157

Arguably, it wouldn't hurt to change both of them to something that does look less arbitrary.

File: web-client/src/presenter/actions/CaseDetail/submitUpdateCaseModalAction.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { state } from 'cerebral';
+
+/**
+ * Updates the case caption and case status
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the get function
+ * @param {object} providers.store the cerebral store object
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/CaseDetail/submitUpdateCaseModalAction.js]: There's no `store` in the jsdoc here, though there is an `applicationContext`.

**Review by kkoskelin — APPROVED** [review]
I could only find one nitpick with a jsdoc that needs adjusting.  Seriously looks like some awesome work.

**Pull request merged**